doc/managing-certificates-for-application-ingress.md

```diff
@@ -0,0 +1,34 @@
+# Managing certificates for application ingress
+
+For each domain in CF:
+
+0. [Make sure SDS is enabled for Istio](https://istio.io/docs/tasks/traffic-management/ingress/secure-ingress-sds/#configure-a-tls-ingress-gateway-using-sds)
+
+1. Create a secret with the certificate and the key in `istio-system` namespace:
+```
+kubectl create secret tls wildcard-apps-example-com-cert \
+    -n istio-system \
+    --cert='path/to/tls.crt' \
+    --key='path/to/tls.key' 
```

Comment:
maybe
```
--cert='path/to/wildcard-apps.example.com.crt' \
--key='path/to/wildcard-apps.example.com.key'
```
just to make it more clear what the user is providing?

Reply:
Updated:
https://github.com/cloudfoundry/cf-k8s-networking/pull/28/commits/2d756e4a49c016411234bef3123b8eb9dee14423